web/styles/subscriptions.css

```diff
@@ -115,7 +115,13 @@ h3.user_group_setting_subsection_title {
     line-height: 1.5;
 }
 
-h4.stream_setting_subsection_title,
+h4.stream_setting_subsection_title {
+    display: inline-block;
+    font-size: 1.35em;
+    font-weight: normal;
+    line-height: 0.5;
+}
```

Comment:
Can you describe how you picked this CSS? A `line-height` of 0.5 is pretty unusual.

Reply:
The line-height was initially set to 1.5. I experimented with value 1 and 0.5 to reduce the space between heading and text. Since 1 was not making much difference, I decided to opt 0.5

Reply:
Instead of changing the `line-height` you can try decreasing the bottom margin for heading.

Reply:
I've set the bottom margin from 10px to 5px. I've also updated the commit history as suggested.